Makefile.custom.mk

```diff
@@ -5,14 +5,14 @@ clean-dry-run: ## dry run for `make clean` - print all untracked files
 .PHONY: clean
 clean: ## Clean the git work dir and remove all untracked files
 	# clean stage
-	git clean -xdf -- test/
+	git clean -xdf -- test/hack/bin
```

Comment:
I think we can keep this line untouched, cleaning the whole `test/` directory sounds fine to me.